client/client.go

```diff
@@ -356,12 +354,11 @@ func (r *NotaryRepository) Publish(getPass passwordRetriever) error {
 
 	// check if our root file is nearing expiry. Resign if it is.
 	if nearExpiry(r.tufRepo.Root) || r.tufRepo.Root.Dirty {
-		passphrase, err := getPass()
```

Comment:
Yeah, but publish makes less sense. I think this is a good thing. Passwords are tied to NotaryRepositories.